### ISSUE TITLE
Schlechte Performance

### ISSUE THREAD

**Issue by ZeroQool007** [ioBroker/ioBroker.socketio#132]
Hallo, der Adapter ist von der Grundidee ziemlich gut, allerdings ist die Performance schlecht. Alleine der Verbindungsaufbau benötigt mehrere Sekunden: Gibt es ein paar Performance Tips? Eingebunden ist es in einer Angular WebApp

Danke

Gruß

**Comment by Apollon77** [comment]
Kann es sein das die "mehreren Sekunden" nur bei der allerersten Verbindung nach Adapterstart relevant sind? Weil wenn das passiert dann wird intern erst einiges initialisiert ...

**Comment by ZeroQool007** [comment]
Ja, das sind teilweise bis zu 10 Sekunden. Wenn man die Smarthome öffnet ist das schon nervig, wenn man erst 10 Sekunden warten muss bis man überhaupt irgendetwas machen kann.

**Comment by Apollon77** [comment]
Nochmal bitte die Frage lesen: Ist es die ERSTE verbindung nach start des socketio Adapters auf Server-Seite oder ist es JEDE verbindung zum Adapter? Was genau tust du? Bitte mal Code tezeigen

**Comment by ZeroQool007** [comment]
Es ist der erste  Aufruf. Code kann ich nachträglich posten. Bin gerade nicht am Rechner.

**Comment by GermanBluefox** [comment]
Gibt es was neues?
